packages/web3/src/index.ts

```diff
@@ -15,6 +15,258 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+/**
+ * This is the main (or 'umbrella') class of the web3.js library.
+ *
+ * ```ts
+ * import { Web3 } from 'web3';
+ *
+ * > Web3.utils //Not exported atm
+ * > Web3.version //Not exported atm
```

Comment:
Will include that in my PR. 

```suggestion
 * > Web3.version 
```